sanskrit_parser/__init__.py

```diff
@@ -1,2 +1,97 @@
-"""
+"""Intro
+======
+
+Sanskrit parser is a python library to help parse Sanskrit input
+
+It provides three main levels of output, in order of increasing complexity:
+  1. *tags* - Morphological analysis of a word
+  2. *sandhi* - Sandhi Split of a phrase
+  3. *vakya* - Morphological Analysis of a sentence (after Sandhi split)
+
+This project is still under development. Please report any issues `here
+<https://github.com/kmadathil/sanskrit_parser/issues>`_.
+
+Command line usage
+==================
+
+The ``sanskrit_parser`` script can be used to view parses, sandhi splits,
+or morphological tags as below.
+
+If the ``--dot`` option is provided, a graph is output in ``.dot`` fomat with
+all the possible morphologies as nodes, and possible relations as
+edges. The valid parses extracted from this graph are also written out
+as ``_parse.dot`` files
+
+
+::
+
+    $ sanskrit_parser vakya astyuttarasyAMdiSi --input SLP1 --dot vakya.dot
+    ...
+    Lexical Split: [asti, uttarasyAm, diSi]
+    ...
+    Parse 0
+    asti=>['as#1', {prATamikaH, praTamapuruzaH, kartari, ekavacanam, law}]
+    diSi=>['diS#2', {ekavacanam, strIliNgam, saptamIviBaktiH}]
+    uttarasyAm=>['uttara#1', {ekavacanam, strIliNgam, saptamIviBaktiH}]
+    ...
+
+    $ dot -Tpng vakya.dot -o vakya.png
+    $ eog vakya.png
+    $ dot -Tpng vakya_parse0.dot -o vakya.png
+    $ eog vakya_parse0.png
+
+
+::
+
+    $ sanskrit_parser sandhi astyuttarasyAMdishidevatAtmA
+    ...
+    Splits:
+    [asti, uttarasyAm, diSi, devatA, AtmA]
+    [asti, uttara, syAm, diSi, devatA, AtmA]
+    [astI, uttarasyAm, diSi, devatA, AtmA]
+    [asti, uttarasyAm, diSi, devatAt, mA]
+    [asti, uttarasyAm, diSi, devata, AtmA]
+    [asti, uttarasyAm, diSi, devat, AtmA]
+    [asti, uttarasyAm, diSi, devatAtmA]
+    [asti, uttarasyAm, diSi, devatA, at, mA]
+    [asti, uttara, syAm, diSi, devat, AtmA]
+    [asti, uttarasyAm, di, Si, devatA, AtmA]
+    -----------
+    ..
+
+::
+
+    $ scripts/sanskrit_parser tags hares
+    ...
+    Morphological tags:
+    ('hf#1', {ekavacanam, viDiliN, prATamikaH, kartari, maDyamapuruzaH})
+    ('hari#1', {ekavacanam, puMlliNgam, paYcamIviBaktiH})
+    ('hari#1', {ekavacanam, paYcamIviBaktiH, strIliNgam})
+    ('hari#1', {zazWIviBaktiH, ekavacanam, puMlliNgam})
+    ('hari#1', {zazWIviBaktiH, ekavacanam, strIliNgam})
+    ('hari#2', {ekavacanam, puMlliNgam, paYcamIviBaktiH})
+    ('hari#2', {ekavacanam, paYcamIviBaktiH, strIliNgam})
+    ('hari#2', {zazWIviBaktiH, ekavacanam, puMlliNgam})
+    ('hari#2', {zazWIviBaktiH, ekavacanam, strIliNgam})
+    ('hf', {ekavacanam, parasmEpadam, viDiliN, maDyamapuruzaH})
+    ('hari', {ekavacanam, puMlliNgam, paYcamIviBaktiH})
+    ('hari', {ekavacanam, strIliNgam, paYcamIviBaktiH})
+    ('hari', {zazWIviBaktiH, ekavacanam, puMlliNgam})
+    ('hari', {zazWIviBaktiH, ekavacanam, strIliNgam})
+    Input String in SLP1: hares
+    ('hf#1', {ekavacanam, viDiliN, prATamikaH, kartari, maDyamapuruzaH})
```

Comment:
Seems to be a duplicate of above (line 67 onwards). I don't get two outputs when I run this command. Copy/paste error?

Reply:
Yes, copy paste. Will fix.